fastdds_statistics_backend.repos

```diff
@@ -2,25 +2,25 @@ repositories:
     foonathan_memory_vendor:
         type: git
         url: https://github.com/eProsima/foonathan_memory_vendor.git
-        version: master
+        version: v1.2.0
     fastcdr:
         type: git
         url: https://github.com/eProsima/Fast-CDR.git
-        version: master
+        version: v1.0.23
     fastdds:
         type: git
         url: https://github.com/eProsima/Fast-DDS.git
-        version: master
+        version: v2.5.0
     fastdds_statistics_backend:
         type: git
         url: https://github.com/eProsima/Fast-DDS-statistics-backend.git
-        version: main
-    fastrtpsgen:
+        version: v0.4.0
+    fastddsgen:
```

Comment:
This should be in another PR to be merged in main branch

Reply:
I have done it in master in this other one: #142

Reply:
Then this is just a remainder:  `0.4.0` branch should be rebased to `main` and then `feature/update_repos` to `0.4.0`

Reply:
Yes! Once all `0.4.0` milestone PRs have been merged it is going to be done.